wrapper.py

```
@@ -0,0 +1,25 @@
+
+def main():
+    arg0 = pathlib.Path(sys.argv[0]).name
+    args = []
+    for arg in sys.argv[1:]:
+        if arg.startswith("-Wp,-D"):
+            args.append(arg.replace("-Wp,-D", "-D", 1))
+        else:
+            args.append(arg)
+
+    if arg0 == "zig-cc":
+        args = ["/mnt/zig", "cc", "-target", "s390x-linux-gnu.2.34"] + args
```

Ruff (0.14.4)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)